### ISSUE TITLE
[Bug Report] Lack syntax color for the light scheme

### ISSUE THREAD

**Issue by laggardkernel** [subtheme-pro/atom-one#3]
**Describe the bug**
The light color scheme lacks lots of syntax definition than the dark one.

- `atom-one/apps/jetbrains/templates/light/scheme.xml`
- `atom-one/apps/jetbrains/templates/dark/scheme.xml`

Can we map the color one to one to build a light scheme from the dark scheme?

**Comment by jontaydev** [comment]
Agreed, the light theme is mostly using default values not the official Atom One colors. Ideally, I would like the [retheme](https://github.com/subtheme-pro/retheme) tool I am working on to automatically pull in the official Atom One colors instead of trying to do this manually. Give me a little bit of time to see if this would be feasible, but worse case I can fork from another repo for now to get this going.
